bin/reth/Cargo.toml

```diff
@@ -53,6 +53,7 @@ reth-payload-primitives.workspace = true
 reth-payload-validator.workspace = true
 reth-basic-payload-builder.workspace = true
 reth-static-file.workspace = true
+reth-static-file-types = { workspace = true, features = ["clap"] }
```

Comment:
we should just activate this feature in `reth-cli-commands`

Reply:
sounds more reasonable, and more little to adjust